pkg/utils/utils.go

```diff
@@ -0,0 +1,36 @@
+// Copyright © 2021 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"os"
+	"path/filepath"
+)
+
+func CacheDir() string {
+	tmpDir, err := os.UserCacheDir()
+	if err != nil {
+		tmpDir = os.TempDir()
+	}
+	return filepath.Join(tmpDir, "tekton")
```

Comment:
Should this be `tkn` to be standardized with config?

Reply:
I like `tkn` better than `tekton`, but I'ld like for other to chim in 🙃 
/cc @tektoncd/cli-maintainers @pierretasci

Reply:
lol, we had a long discussion about where to put th ~/.config/tkn or tekton before with a survey and all.. I was in favor of `.tkn`, i think it would be best to have it the same as ".tkn"

Reply:
+1 to that.

Reply:
swapped to tkn